spec/workers/commit_monitor_handlers/commit_range/rubocop_checker/data/with_results_with_no_offenses/no_offenses.rb

```diff
@@ -1,2 +1,5 @@
 class NoOffenses
+  def initialize
+    @x = 1
+  end
```

Comment:
This was needed because of https://docs.rubocop.org/rubocop/1.13/cops_lint.html#lintemptyclass